builtin/logical/pki/cert_util.go

```diff
@@ -80,7 +80,7 @@ func getFormat(data *framework.FieldData) string {
 }
 
 // fetchCAInfo will fetch the CA info, will return an error if no ca info exists.
-func fetchCAInfo(ctx context.Context, b *backend, req *logical.Request, issuerRef string) (*certutil.CAInfoBundle, error) {
+func fetchCAInfo(ctx context.Context, b *backend, req *logical.Request, issuerRef string, issuance bool, crl bool) (*certutil.CAInfoBundle, error) {
```

Comment:
This feels like as we add more usages in the future could just expand to be really easy to pass in the wrong params. 

Could we instead introduce, internally, an iota of usage constants, and passing them them in kinda like the replication [HasState](https://github.com/hashicorp/vault/blob/main/sdk/helper/consts/replication.go#L147) flagging system?

Reply:
:) Oops, well I suppose you know my opinion. 

I think it would make things easier if we pushed the bitset all the way to the entry as well, I believe it will make the comparison easier at the sacrifice of making possibly a future patch update a little more complicated (fetch, update and set)

Reply:
I agree.  Positional arguments of the same type are fragile.